Content.Shared/Ninja/Systems/DashAbilitySystem.cs

```diff
@@ -80,6 +85,14 @@ private void OnDash(Entity<DashAbilityComponent> ent, ref DashEvent args)
             return;
         }
 
+        // Check if the user is BEING pulled, and escape if so
+        if (TryComp<PullableComponent>(user, out var pull) && _pullingSystem.IsPulled(user, pull))
+            _pullingSystem.TryStopPull(user, pull);
+
+        // Check if the user is pulling anything, and drop it if so
+        if (TryComp(user, out PullerComponent? puller) && TryComp(puller.Pulling, out PullableComponent? pullable))
```

Comment:
```suggestion
        if (TryComp<PullerComponent>(user, out var puller) && TryComp<PullableComponent>(puller.Pulling, out var pullable))
```
maybe make it a little more consistent for better readability